apps/docs/src/app/core/component-docs/calendar/examples/calendar-disabled-navigation-buttons-example/calendar-disabled-navigation-buttons-example.component.ts

```diff
@@ -0,0 +1,52 @@
+import { Component } from '@angular/core';
+import { FdDate } from 'libs/core/src/lib/datetime';
+import { FdCalendarView } from 'libs/core/src/lib/calendar';
+import { CalendarCurrent } from 'libs/core/src/lib/calendar/models/calendar-current';
+
+@Component({
+    selector: 'fd-calendar-disabled-navigation-buttons-example',
+    templateUrl: 'calendar-disabled-navigation-buttons-example.html'
+})
+export class CalendarDisabledNavigationButtonsExampleComponent {
+    /**
+     * Function used to disable previous button in the calendar header.
+     * @param date selected date
+     * @param currentlyDisplayedDate currently displayed date
+     * @param activeView current view of calendar
+     * will disable choosing dates before 2022 also if we are in year view or if we selected January
+     */
+    previousButtonDisableFunction(
+        date: FdDate,
+        currentlyDisplayedDate: CalendarCurrent,
+        activeView: FdCalendarView
+    ): boolean {
+        if (date.month === 1) {
+            return true;
+        }
+        if (activeView === 'year') {
+            return true;
+        }
+        return currentlyDisplayedDate?.year < 2022;
+    }
+
+    /**
+     * Function used to disable next button in the calendar header.
+     * @param date selected date
+     * @param currentlyDisplayedDate currently displayed date
+     * @param activeView current view of calendar
+     *  will disable choosing dates after 2023, also if we are in year view or if we selected January
+     */
+    nextButtonDisableFunction(
+        date: FdDate,
+        currentlyDisplayedDate: CalendarCurrent,
+        activeView: FdCalendarView
+    ): boolean {
+        if (date.month === 1) {
+            return true;
+        }
+        if (activeView === 'year') {
+            return true;
+        }
+        return currentlyDisplayedDate?.year > 2023;
+    }
```

Comment:
sorry, I didn't notice this before: don't use hardcoded dates, because this example will stop working in 2024
```suggestion
     * will disable choosing dates before current year also if we are in year view or if we selected January
     */
    previousButtonDisableFunction(
        date: FdDate,
        currentlyDisplayedDate: CalendarCurrent,
        activeView: FdCalendarView
    ): boolean {
        if (date.month === 1) {
            return true;
        }
        if (activeView === 'year') {
            return true;
        }
        return currentlyDisplayedDate?.year < new Date().getFullYear();
    }

    /**
     * Function used to disable next button in the calendar header.
     * @param date selected date
     * @param currentlyDisplayedDate currently displayed date
     * @param activeView current view of calendar
     *  will disable choosing dates after next year, also if we are in year view or if we selected January
     */
    nextButtonDisableFunction(
        date: FdDate,
        currentlyDisplayedDate: CalendarCurrent,
        activeView: FdCalendarView
    ): boolean {
        if (date.month === 1) {
            return true;
        }
        if (activeView === 'year') {
            return true;
        }
        return currentlyDisplayedDate?.year > (new Date().getFullYear() + 1);
    }
```